lib/msal-common/src/authority/AuthorityOptions.ts

```diff
@@ -12,4 +12,22 @@ export type AuthorityOptions = {
     cloudDiscoveryMetadata: string;
     authorityMetadata: string;
     azureRegionConfiguration?: AzureRegionConfiguration;
+    azureCloudInstance?: AzureCloudInstance;
 };
+
+export enum AzureCloudInstance {
+    // AzureCloudInstance is not specified.
+    None,
+
+    // Microsoft Azure public cloud. Maps to https://login.microsoftonline.com
+    AzurePublic,
+
+    // Microsoft Chinese national cloud. Maps to https://login.chinacloudapi.cn
+    AzureChina,
```

Comment:
```suggestion
    AzureChina = "https://login.chinacloudapi.cn",
```

Reply:
Same as above, keeping this as is for consistency.